app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java

```diff
@@ -974,6 +975,11 @@ private void setHeightThumbnail() {
 
     private void showContent() {
         contentRootLayoutHiding.setVisibility(View.VISIBLE);
+        showDescriptionOnLoad = PreferenceManager.getDefaultSharedPreferences(activity)
```

Comment:
Since you use this variable only once and only here, it should be a local field
```suggestion
        final boolean showDescriptionOnLoad = PreferenceManager.getDefaultSharedPreferences(activity)
```

Reply:
right. done